quinn-h3/src/client.rs

```diff
@@ -1,3 +1,72 @@
+//! Client implementation for the HTTP/3 protocol.
+//!
+//! # Overview
+//!
+//! Start by constructing a [`Client`] endpoint with a [`Builder`], then [`connect()`] to a server.
+//! Before being able to issue requests, an handshake phase will need to be completed, waiting for
+//! [`Connecting`] to resolve. You can also choose to try to issue your first request even before
+//! the hanshake succeeds, using the [`0-RTT`] feature.
+//!
+//! Once a [`Connection`] is up and running, you can start sending [`Request`]s via
+//! [`Connection::send_request()`]. After this, a [`RecvResponse`] and a [`BodyWriter`] future are
+//! made available for, respectively, receiving the response, and sending the request's body, if any.
+//!
+//! [`RecvResponse`] represents the response's header reception and decoding. It will yeild
```

Comment:
```suggestion
//! [`RecvResponse`] represents the response's header reception and decoding. It will yield
```

Would be nice if you remember this spelling. 😛